src/main/java/org/hmhb/organization/DefaultOrganizationService.java

```diff
@@ -98,6 +112,11 @@ public Organization save(
             organization.setUpdatedBy(null);
             organization.setUpdatedOn(null);
         } else {
+
+            if (!authorizationService.isAdmin()) {
```

Comment:
If feels a little weird allowing anyone to create an org in phase1 (only admins are inputting the data -- so I could easily require admin for org creation), but that is the eventual requirement.